JSDemos/Demos/List/ListSelection/description.md

```diff
@@ -1 +1,9 @@
-The **List** UI component offers three item selection modes: _&laquo;all&raquo;_, _&laquo;single&raquo;_ and _&laquo;multi&raquo;_. Note that when the current mode is _&laquo;all&raquo;_, the **List** allows a&nbsp;user to&nbsp;select all items at&nbsp;once by&nbsp;clicking the _Select All_ check box. The **selectAllMode** property allows you to&nbsp;decide whether all items should be&nbsp;selected across all pages or&nbsp;within the currently rendered pages only.
\ No newline at end of file
+The **List** offers three item selection modes: *"single"*, *"multiple"*, and *"all"*.  To enable item selection, pass a mode name to the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property. If you set this property to *"none"* (default setting), item selection is disabled. In this example, you can try out all the selection modes. Use the Selection Mode drop-down menu to switch between them.
```

Comment:
```suggestion
There are three **List** item selection modes: *"single"*, *"multiple"*, and *"all"*.  To enable item selection, pass the mode name to the [selectionMode](/Documentation/ApiReference/UI_Components/dxList/Configuration/#selectionMode) property. If you set this property to *"none"* (default setting), item selection is disabled. In this example, you can try out all the selection modes. Use the drop-down Selection Mode menu to switch between modes.
```

Reply:
What do you mean when you say "In this example, you can try out all the selection modes"? How can you try it out in an example?

Reply:
When you select a mode from the dropdown menu, the change is reflected in the List. However, to see the change, you have to try and select items. Please see the demo linked in the Trello card for examples

Reply:
No, I mean why do you refer to a demo as an example? If you can switch selection modes in a demo, do not call it an example.